tests/index.html

```diff
@@ -10,11 +10,21 @@
   <div id="qunit"></div>
   <div id="qunit-fixture"></div>
 
+  <script src="/sdk-testing.js"></script>
+  <!-- this needs thought but it brings in polyfills for the amd build :( -->
+  <script src="shopify-buy.umd.polyfilled.js"></script>
+  <script src="/qunit-shim.js"></script>
+  <script src="/shopify-buy.amd.js"></script>
+  <script src="/eslint-tests.js"></script>
+  <script src="/tests.js"></script>
```

Comment:
also curious about this, I couldn't find a clean way to have testem run separate top-level "suites" without using html files